### PR TITLE
Chapter 3: Mostly static pages

### PR DESCRIPTION
**Pull request by rainmaize**
60761132_2314669735520043_5014731487560335360_n

### CODE REVIEW

**Comment by rainmaize** [comment]
ready

**Comment by phamvanbk11** [comment]
commented

**Comment by rainmaize** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
+gem 'puma', '~> 3.11'
+gem 'rails', '~> 5.2.3'
+gem 'sass-rails', '~> 5.0'
+gem 'sqlite3'
+gem 'uglifier', '>= 1.3.0'
 
-gem "bootsnap", ">= 1.1.0", require: false
+gem 'coffee-rails', '~> 4.2'
+gem 'jbuilder', '~> 2.5'
+gem 'turbolinks', '~> 5'
```
[the remark this answers is not in the document]
Comment (rainmaize) [reply Gemfile]: đã fix

File: app/views/static_pages/about.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
+<h1>About</h1>
+<p>
+  The <a href="https://www.railstutorial.org/"><em>Ruby on Rails
+  Tutorial</em></a> is a
+  <a href="https://www.railstutorial.org/book">book</a> and
+  <a href="http://screencasts.railstutorial.org/">screencast series</a>
+  to teach web development with
+  <a href="http://rubyonrails.org/">Ruby on Rails</a>.
+  This is the sample application for the tutorial.
```
[the remark this answers is not in the document]
Comment (rainmaize) [reply app/views/static_pages/about.html.erb]: đã fix

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**